laravel/database/seeds/ConfigTableSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use App\Models\Config;
+use Illuminate\Database\Seeder;
+
+class ConfigTableSeeder extends Seeder
```

Comment:
This is an example database `seeder`. This populates a table with default data and would typically be run only on initial installation on a real server to 'seed' it with system defaults.

```
vagrant ssh
cd /vagrant/laravel
php artisan db:seed
```

During development you'll probably do this after every VM destruction or `php artisan db:fresh` to get things back to a known state.

To make these seed classes use the artisan command

```
vagrant ssh
cd /vagrant/laravel
php artisan make:seeder config_table_seeder
```